doc/classes/ResourceLoader.xml

```diff
@@ -67,9 +67,11 @@
 			<return type="ResourceInteractiveLoader" />
 			<argument index="0" name="path" type="String" />
 			<argument index="1" name="type_hint" type="String" default="&quot;&quot;" />
+			<argument index="2" name="no_cache" type="bool" default="false" />
 			<description>
 				Starts loading a resource interactively. The returned [ResourceInteractiveLoader] object allows to load with high granularity, calling its [method ResourceInteractiveLoader.poll] method successively to load chunks.
 				An optional [code]type_hint[/code] can be used to further specify the [Resource] type that should be handled by the [ResourceFormatLoader]. Anything that inherits from [Resource] can be used as a type hint, for example [Image].
+				If [code]no_cache[/code] is [code]true[/code], the resource cache will be bypassed and the resource will be loaded anew. Otherwise, the cached resource will be returned if it exists.
```

Comment:
Needs comma after `bypassed`.

Reply:
I copy pasted the line from `load()`'s description just above it. Shoud I change both?

Reply:
Yeah, they would both be incorrect then. `If...true[/code],` is a dependent phrase, `the resource....bypassed` is an independent phrase, and then `the resource...anew` is a second independent phrase. Two independent phrases have to be conjoined with a conjunction preceded by a comma, so both paragraphs are missing the comma.